Makefile

```diff
@@ -19,3 +19,12 @@ lint: prometheus_alert_rules.yaml
 
 clean:
 	rm -rf prometheus_alert_rules.yaml
+
+generate_k8s:
+	cd extras; cp manifests/prometheus-rules.yaml .; jb install; ./build.sh example.jsonnet; cd -
+	cmp -s extras/prometheus-rules.yaml extras/manifests/prometheus-rules.yaml; \
+	RETVAL=$$?; \
+	if [ $$RETVAL -eq 0 ]; then \
```

Comment:
Should be "not equal to 0" ??

Reply:
Yes, that's a mistake. Will correct that.